webpack.config.js

```diff
@@ -179,7 +173,7 @@ if ( hasReactFastRefresh ) {
 
 const sassTest = /\.(sc|sa)ss$/;
 const updatedSassOptions = {
-	sourceMap: process.env.NODE_ENV === 'production',
+	sourceMap: ! isProduction,
```

Comment:
Yes, it's the "Generate source map of SCSS in development." item in the PR description.

Originally, the config here is the opposite of the `css-loader` in the [default webpack config](https://github.com/WordPress/gutenberg/blob/%40wordpress/scripts%4012.5.0/packages/scripts/config/webpack.config.js#L31-L33) in `wp-script`, so the source map generated during development does not point to the source scss files. And the production build will not generate any source map in the end because of the `css-loader` config. So I think it's more effective to change it to a consistent config so that the source map can point to the correct scss source files during development.